modules/mono/editor/GodotSharpTools/Build/BuildSystem.cs

```diff
@@ -196,7 +196,7 @@ private string BuildArguments(string loggerAssemblyPath, string loggerOutputDir,
 
             foreach (string customProperty in customProperties)
             {
-                arguments += " \"/p:" + customProperty + "\"";
+                arguments += " /p:" + customProperty;
```

Comment:
Why remove the double quotes?

Reply:
When providing paramters that have multiple values separated by `;` we need to surround them with double quotes. Can't do that if the parameter itself is completely surrounded by double quotes already.